lib/load_brocfile.js

```diff
@@ -4,23 +4,76 @@ const path = require('path');
 const findup = require('findup-sync');
 const esmRequire = require('esm')(module);
 
+/**
+ * Require a brocfile via either ESM or TypeScript
+ *
+ * @param {String} brocfilePath The path to the brocfile
+ * @returns {*}
+ */
+function requireBrocfile(brocfilePath) {
+  let brocfile;
+
+  if (brocfilePath.match(/\.ts$/)) {
+    const configPath = findup('tsconfig.json', {
+      nocase: true,
+    });
+
+    let options;
+
+    if (configPath) {
+      options = { project: configPath };
+    } else {
+      options = {
```

Comment:
If no `tsconfig.json` file is found (or maybe even if it is), would it make sense to use `ts-node` in `transpileOnly` mode? It's a bit faster since it skips typechecking, and you don't risk enabling config options that will break the user.

Reply:
honestly not sure, happy to take advice on this for recommended default (can always be changed with tsconfig)

Reply:
typescript already has some defaults it falls back to in the absence of a `tsconfig.json`. We may want to consider remaining consistent with that, in order to avoid introducing ts-node as a dependency, and confusing those who expect "normal" behavior from config-less typescript compilation

Reply:
Removed the tsconfig loading, defaults to the project dir tsconfig.json anyway, and works out of the box.